lib/logo.sh

```diff
@@ -1,21 +1,33 @@
 #!/bin/bash
 
-_logo(){
+_logo_display(){
 cat <<EOF
         ____  __   __  ___     ___ _
        / __ \/ /  /  |/  /_ __/ __| |_
-      / /_/ / _ \/ /|_/ / // /\__ \ . \ 
-      \____/_//_/_/  /_/\_, / |___/_||_|
+      / /_/ / _ \/ /|_/ / // /\__ \ . \  
+      \____/_//_/_/  /_/\_, / |___/_||_| [] [] []
                        /___/
+
 EOF
 }
 
-_des(){
-  _logo
+_logo(){
+  if [ "$(checkcmd lolcat)" = "1" ]
+  then
+    # play 3 time an bell sound
+    for i in {1..3}; do
+      echo -e "\a"
```

Comment:
I don't know why I got `")` output in my MacBook.

Could you try `tput bel` command on your os? This can run on my Mac.